packages/apps/human-app/frontend/src/shared/helpers/regex.ts

```diff
@@ -3,9 +3,8 @@ export const password8Chars =
 export const passwordUppercase = /(?=.*[A-Z])/;
 export const passwordLowercase = /(?=.*[a-z])/;
 export const passwordNumeric = /(?=.*[0-9])/;
-export const passwordSpecialCharacter =
-  /(?=.*[$^*.[\]{}()?"!@#%&/\\,><':;|_~`=+-])/;
+export const passwordSpecialCharacter = /[!@#$%^&+=*]/; // allowed characters: @#$%^&+=!*]
 
-// this is sum of all above regular expressions
+// this is sum of all above regular expressions that matches with reputation oracle 'auth.dto.ts'
 export const passwordRegex =
-  /^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[$^*.[\]{}()?"!@#%&/\\,><':;|_~`=+-])[A-Za-z0-9$^*.[\]{}()?"!@#%&/\\,><':;|_~`=+-]{8,}$/;
+  /^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&+=*])(?=.{8,})/;
```

Comment:
I think we don't need this since the only restriction we'll have for the password is length.

Reply:
done